### PR TITLE
More properly read in package.json files, kinda

### LINKED ISSUE #120: get repo from package.json
with #119 i fixed #117 but i am pretty sure deserializing a package.json is gonna have issues. here's how you can specify a repo in package.json: https://docs.npmjs.com/cli/v9/configuring-npm/package-json#repository

### PR DESCRIPTION
**Pull request by Gankra**
~~We can defer to a proper library for the definitions~~ We can write out own definitions and just pull out the parts we want *but* the repository field is just super whack and complex so it's probably not a URL. If you have some npm project that you Definitely Want To Work I can roll up my sleeves and actually iterate on the behaviour.

Fixes #120

### CODE REVIEW

**Comment by SaraVieira** [comment]
Since you are here... Can you please also return the license from it?

**Comment by SaraVieira** [comment]
I know this person wanted to use Oranda
https://github.com/jsx-eslint/eslint-plugin-react/blob/master/package.json

File: tests/mod.rs [hunk 1]
```diff
@@ -63,6 +63,7 @@ fn it_loads_a_js_project_config() {
             r#"
 {
     "name": "axo",
+    "version": "0.1.0",
```
Comment (ashleygwilliams) [inline tests/mod.rs:66]: should we add repo to this as well?

Comment (Gankra) [reply tests/mod.rs:66]: done

**Review by ashleygwilliams — APPROVED** [review]

**Pull request merged**